src/main/scala/chiffre/passes/FaultInstrumentation.scala

```diff
@@ -159,19 +153,25 @@ class FaultInstrumentation(compMap: Map[String, Seq[(ComponentName, String, Clas
                 ComponentName(s"$defi.io.scan.$s",
                               ModuleName(m.name, CircuitName(c.main))) )
 
+            val wt = classOf[firrtl.passes.wiring.WiringTransform]
+            val st = classOf[chiffre.passes.ScanChainTransform]
+
+            val faulty = DefWire(NoInfo, rename, t)
+            val data = fromBits(WRef(faulty), toExp(s"$defi.io.out")) match {
+              case Block(stmts: Seq[Statement]) => stmts :+ Connect(NoInfo, toExp(s"$defi.io.in"),
+                         toBits(WRef(comp.name, t, RegKind, UNKNOWNGENDER)))
+              case _ => Seq.empty[Statement]
```

Comment:
The default case is unreachable with the current implementation of `firrtl.fromBits`. While we could throw an exception, I think it's fine to remove the default case and deal with the noisy match error if `firrtl.fromBits` ever changes. With this as it is, if we ever hit the default case then we'd wind up with an unconnected fault injector (which may precipitate a FIRRTL error or may be silently broken).